test/test-helpers.ts

```diff
@@ -26,7 +25,7 @@ export function getExpectedObject(exampleObject: any, overrideProperties = {}) {
   var expectedObject = { ...exampleObject, ...overrideProperties };
 
   if ("id" in expectedObject)
-    expectedObject.id = expect.stringMatching(/^[0-9a-z-]+$/);
+    expectedObject.id = expect.stringMatching(/^[0-9a-z-_]+$/);
```

Comment:
Same comment as before regarding the `_` which should not appear in uuid4.

Reply:
Closing after comment in https://github.com/onfido/onfido-node/pull/108#discussion_r1530060308

Reply:
It contains `_`. Maybe, I should add a custom validator for tasks?

```
{
        "created_at": "2024-03-19T09:59:51Z",
        "id": "upload_document_photo_024a2",
        "task_def_id": "upload_document_photo",
        "task_def_version": null,
```

Reply:
As you prefer, I don't have a strong opinion on that.